### DISCUSSION TITLE
next/dynamic custom loading component: pass props through

### DISCUSSION THREAD

**Discussion by tremby** [vercel/next.js#20261] [Help]
Documented here: https://nextjs.org/docs/advanced-features/dynamic-import#with-custom-loading-component

It would be very useful for my current use case if the props which are passed through to the *loaded* version of the dynamic component are also passed through to the custom loading component.

My particular use case is for a dynamically-loaded icon. I want the placeholder to be the same size, which means knowing what the `size` prop passed in was. At the moment I can't access that from the custom loading component.

**Comment by OmarJaroudi** [comment]
Bump on this, you can do this with React.lazy and suspense, it only makes sense to have it available with next/dynamic
